### PR TITLE
add newsflash-greader to README

### PR DESCRIPTION
**Pull request by Stunkymonkey**
Is this the way I should update the README?

Changes proposed in this pull request:

- add newsflash greader.

Pull request checklist:

- [x] clear commit messages
- [ ] code manually tested
- [ ] unit tests written (optional if too hard)
- [x] documentation updated

[Additional information can be found in the documentation](https://github.com/FreshRSS/FreshRSS/tree/edge/docs/en/developers/04_Pull_requests.md).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -113,16 +113,17 @@ and [Fever API](https://freshrss.github.io/FreshRSS/en/users/06_Fever_API.html)
...
 | [News+](https://github.com/noinnion/newsplus/blob/master/apk/NewsPlus_202.apk) with [Google Reader extension](https://github.com/noinnion/newsplus/blob/master/apk/GoogleReaderCloneExtension_101.apk) | Android | [Partially](https://github.com/noinnion/newsplus/blob/master/extensions/GoogleReaderCloneExtension/src/com/noinnion/android/newsplus/extension/google_reader/) | 2015       | GReader | ✔️             | ⭐⭐⭐       | ✔️                    | ✔️                 | ✔️         | ✔️     | ✔️       | ✔️           |
-| [FeedMe](https://play.google.com/store/apps/details?id=com.seazon.feedme)*            | Android     | ➖                                                            | ✔️✔️                   | GReader          | ✔️            | ⭐⭐      | ➖                             | ➖                  | ✔️         | ✓     | ✔️       | ✔️           |
+| [FeedMe](https://play.google.com/store/apps/details?id=com.seazon.feedme)*            | Android     | ➖                                                            | ✔️✔️                   | GReader          | ✔️            | ⭐⭐      | ➖                             | ➖                  | ✔️         | ✔️     | ✔️       | ✔️           |
```
Comment (Alkarex) [inline README.md:116]: Is there now a way in FeedMe to show the labels? (and showing the articles with a given label)
That is what was missing to get the full ✔️

**Comment by Alkarex** [comment]
Duplicate of https://github.com/FreshRSS/FreshRSS/pull/4650 ?
I just need to find a moment to get it to run to give it a try

File: README.md [hunk 2]
```diff
@@ -113,16 +113,17 @@ and [Fever API](https://freshrss.github.io/FreshRSS/en/users/06_Fever_API.html)
...
-| [Fluent Reader Lite](https://hyliu.me/fluent-reader-lite/)                            | Android, iOS| [✔️](https://github.com/yang991178/fluent-reader-lite)        | ✔️✔️                   | GReader, Fever   | ✔️            | ⭐⭐⭐    | ➖                             | ➖                  | ✓         | ➖     | ➖       | ➖           |
-| [FocusReader](https://play.google.com/store/apps/details?id=allen.town.focus.reader)  | Android     | ➖                                                            | ✔️✔️                   | GReader          | ✔️            | ⭐⭐⭐    | ➖                             | ➖                  | ✔️         | ➖     | ✓       | ✔️           |
+| [Fluent Reader Lite](https://hyliu.me/fluent-reader-lite/)                            | Android, iOS| [✔️](https://github.com/yang991178/fluent-reader-lite)        | ✔️✔️                   | GReader, Fever   | ✔️            | ⭐⭐⭐    | ➖                             | ➖                  | ✔️         | ➖     | ➖       | ➖           |
```
Comment (Alkarex) [inline README.md:119]: Did you check the favourites synchronisation in Fluent Reader Lite? Last time I did, there was only partial support

File: README.md (hunk 1 above)
Comment (Stunkymonkey) [reply README.md:116]: I just thought the check mark was wrong on purpose.

**Pull request closed without merging**